File: 프로그래머스/lv1/12901. 2016년/2016년.js
function solution(a, b) {
    let answer = '';
    let day = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
    let bowl = b - 1;
    let dayOfTheWeek = ["FRI","SAT","SUN","MON","TUE","WED","THU"];
        for(let i = 0; i < a - 1; i++){
            bowl += day[i];
        }
    return answer = dayOfTheWeek[(bowl % 7)];
}

File: 프로그래머스/lv1/12901. 2016년/2016년.java
class Solution {
    public String solution(int a, int b) {
        String answer = "";
        int[] day = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
        int bowl = b - 1;
        String[] dayOfTheWeek = {"FRI","SAT","SUN","MON","TUE","WED","THU"};
        for(int i = 0; i < a - 1; i++){
            bowl += day[i];
        }
        System.out.println("bowl = " + bowl);
        System.out.println("bowl % 7 = " + (bowl % 7));
        return answer = dayOfTheWeek[(bowl % 7)];
    }
}